include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -547,8 +535,8 @@ struct __result_and_scratch_storage_impl : __result_and_scratch_storage_base
     bool __supports_USM_device;
 
     // Only use USM host allocations on L0 GPUs. Other devices show significant slowdowns and will use a device allocation instead.
-    inline bool
-    __use_USM_host_allocations(sycl::queue __queue)
+    static bool
```

Comment:
Its a little strange to me to have these two private `static` functions.  Can we just call these from within the constructor and use the queue member field rather than passing in a queue as an argument?

(I understand this is not the purpose of this PR, but marking them `static` shows how odd this is...)

Reply:
Fixed.